mod_ci/controllers.py

```diff
@@ -866,6 +868,57 @@ def show_maintenance():
     }
 
 
+@mod_ci.route('/blocked_users')
+@login_required
+@check_access_rights([Role.admin])
+@template_renderer()
+def blocked_users():
+        blocked_users = BlockedUsers.query.order_by(BlockedUsers.userID)
+
+        usernames = {}  # Initialize usernames dictionary
+        for instance in blocked_users:
+            # Set values as an error variable
+            usernames[instance] = 'Error, cannot get username'
```

Comment:
This will assign a full model as key. I'd prefer seeing the id's of the users instead.

Besides, this can be done in a more pythonic way:

```
usernames = {u.userID: 'Error, cannot get username' for u in blocked_users}
```

Reply:
Ah, list comprehensions right

Reply:
Yes